### PR TITLE
Enhance instructors exam checklist table

### PR DESCRIPTION
**Pull request by nicolasruscher**
### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.
- [x] Client: I followed the [coding and design guidelines](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/client.html).
- [x] Client: I added multiple integration tests (Jest) related to the features (with a high test coverage)
- [x] Client: I documented the TypeScript code using JSDoc style.
- [x] Client: I added multiple screenshots/screencasts of my UI changes
- [x] Client: I translated all the newly inserted strings into German and English

### Description

We enhanced the instructors exam checklist table by reorganizing the way that the exercise group informations are presented. We combined checks into one table. If a group has problems, a orange warning will appear next to the group.

### Steps for Testing

1. Log in to Artemis
2. Navigate to Course Administration -> exam -> checklist
3. Check if the informations are displayed correctly. 

### Test Coverage

ExamChecklistExerciseGroupTableComponent: 95%

### Screenshots

Bildschirmfoto 2021-03-10 um 17 49 30

### CODE REVIEW

**Comment by tiffca** [comment]
this didn't work for me on existing exams, does this only work with new exams? 

image

**Review by tiffca — APPROVED** [review]
Tested successfully

**Review by SimonEntholzer — APPROVED** [review]
Tested successfully on TS1, the code looks good too :D

File: src/main/webapp/app/exam/manage/exams/exam-checklist-component/exam-checklist-check/exam-checklist-check.component.html [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <fa-icon *ngIf="checkAttribute" [icon]="'check-circle'" style="color: #28a745"></fa-icon>
-<fa-icon *ngIf="!checkAttribute" [icon]="'times'" style="color: #ca2024"></fa-icon>
+<fa-icon *ngIf="!checkAttribute" [icon]="'times'" style="color: #ca2024; font-size: 150%"></fa-icon>
```
Comment (ivanchimeno) [inline src/main/webapp/app/exam/manage/exams/exam-checklist-component/exam-checklist-check/exam-checklist-check.component.html:2]: You can use em

**Review by ivohashamov — APPROVED** [review]
Works as expected, tested on ts1

**Review by SimonEntholzer — APPROVED** [review]
Tested in testing session today, still works so re-approve :D

**Pull request merged**